src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/TermQueryExplainIT.java

```diff
@@ -358,6 +358,18 @@ public void testNestedSingleOrderBy() throws IOException {
         assertThat(result, containsString("\"nested\":{\"path\":\"projects\""));
     }
 
+    @Test
+    public void testNestedIsNotNull() throws IOException {
```

Comment:
Could we add another IT case for real execution along with this explain testing? And then this one can be renamed to `testNestedIsNotNullExplain`.

Reply:
Add IT. Done.